client/src/ipfsUploader.js

```diff
@@ -0,0 +1,13 @@
+import { create } from 'ipfs-http-client'
+
+export const uploadToIpfs = async (file) => {
+    const client = create('https://localhost:5001')
+    try {
+        console.log("IPFS: START uploading")
```

Comment:
Better print out the file name that you're uploading, unless the client.add call prints it.